DCP_Tutorial.md

```diff
@@ -0,0 +1,135 @@
+3) Your workflow will move through `Prep`, `Analysis`, and `Submit` stages. In the final stage, it will interact with the ingest API and HCA cli to upload the workflow output files within a submission. The spreadsheet to JSON conversion is skipped, as the analysis component directly deposits JSON files via the ingest API. The workflow can take up to 2-3 hours. Go do some work!
+4) After the workflow completes and the corresponding ingest envelope has a state of `Completed`, you should now be able to retrieve your analysis results from the data store. Lets switch it up and use some python and the HCA dss python bindings to find your bundle. Copy the python below to a file on your local machine. Invoke it with `python script.py --primary-bundle-uuid {YOUR_PRIMARY_BUNDLE_UUID}`. This script will print out the bundle FQID of your analysis results.
+```
+#!/usr/bin/env python
+import argparse
+
+from hca.dss import DSSClient
+
+
+def find_secondary_bundle_fqids(client, replica, primary_bundle_uuid):
+    q = {
+        'query': {
+            'match': {
+                "files.analysis_process_json.input_bundles": primary_bundle_uuid
+            }
+        }
+    }
+    fqids = execute_query(client, replica, q)
+    results = list()
+    for secondary_fqid in fqids:
+        results.append(secondary_fqid)
+    print(results)
+
+
+def execute_query(client, replica, query):
+    resp = client.post_search(replica=replica, es_query=query)
+    fqids = [hit['bundle_fqid'] for hit in client.post_search.iterate(replica=replica, es_query=query)]
+    assert len(fqids) == resp['total_hits']
+    return fqids
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument("--replica", default="aws", choices=["aws", "gcp"])
+    parser.add_argument("--primary-bundle-uuid")
+    args = parser.parse_args()
+    dss_client = DSSClient(swagger_url=f"https://dss.staging.data.humancellatlas.org/v1/swagger.json")
+    find_secondary_bundle_fqids(dss_client, args.replica, args.primary_bundle_uuid)
+```
+5) Lets go ahead and download the analysis bundle using the HCA cli. This step will download the bundle into a directory named after bundle UUID.
+        hca dss download --bundle-uuid {BUNDLE_UUID_PRINTED_FROM_SCRIPT) --replica {AWS OR GCP}
+6) Go ahead and open up some of the files in the directory to get a feel for what is contained within an analysis bundle, including an individual gene x cell matrix.
+
+#### Phase 4: Viewing project in the Browser
+**Components:** Data Store, Azul, Data Browser  
+**Description:** In this phase, you will be looking at your project in the data browser!
+1) This is more of an informational step. Just as analysis has a subscription to the data store, azul (the browser backend) has a subcription to all bundles in the data store. Azul will then do a partial download of the bundle and index it based on the facets that are made available for search and exploration in the data browser.
```

Comment:
```suggestion
1) This is more of an informational step. Just as Analysis has a subscription to the Data Store, Azul (the Data Browser backend service) has a subscription to all bundles in the Data Store. When Azul is notified about a new bundle, it will do a download of the metadata in the bundle and index it based on the facets that are made available for search and exploration in the Data Browser.
```

Reply:
Accepting suggestion. Thanks!